stats/rpc_constants.textpb

```diff
@@ -0,0 +1,770 @@
+# Constants for collecting rpc stats.
```

Comment:
Need a longer, more descriptive comment here. Should summarize the rest of the file.